.env

```diff
@@ -0,0 +1 @@
+jwtSecret=Aysha1911
```

Comment:
It's cool to use a `.env` file, but it should never be committed as part of the repo, because it usually contains sensitive information. 
https://www.npmjs.com/package/dotenv#should-i-commit-my-env-file

It is, however, often common practice to create a `.env.example` file, which might just have the keys but no (or example) values, e.g.
```
jwtSecret=
DB_USER=
DB_PASS=
```
When someone clones their repo, they can look at the example or copy it to a .env `> cp .env.example .env` locally.

Do you know what the `.gitignore` file is? I can help you remove this if you need help.

Reply:
I think I definitely do with some help here.Thank you,